README.md

```diff
@@ -14,9 +14,11 @@ Davincibox has **not** been tested with DaVinci Resolve Studio. See [#26](https:
 
 ## Requirements
 
-You will need `distrobox` or `toolbox`. Distrobox is highly recommended for ease of use.
+You will need [`distrobox`](https://github.com/89luca89/distrobox) or [`toolbox`](https://github.com/containers/toolbox). Distrobox is highly recommended for ease of use.
 
-If using an Nvidia GPU, you will also need the `nvidia-container-runtime` package installed to your system.
+If using an Nvidia GPU, you will also need the `nvidia-cuda-toolkit` package installed to your system.
```

Comment:
The GitHub repository for [nvidia-container-toolkit](https://github.com/NVIDIA/nvidia-container-toolkit) states "Note that you **do not need to install the CUDA Toolkit** on the host system, but the NVIDIA driver needs to be installed"

I can't test it myself since I have an AMD GPU, but it sounds like just having the NVIDIA driver installed on its own should work now?

Or, if it's referring to `nvidia-container-toolkit` itself, that seems to be a different package than `nvidia-cuda-toolkit`. See https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html